src/containers/checkout/checkout-form/checkout-form.js

```diff
@@ -0,0 +1,216 @@
+                />
+                {touched.code && errors.code && (
+                  <div data-cy='code-error' className={styles.error}>
+                    {errors.code}
+                  </div>
+                )}
+                <TextField
+                  size='small'
+                  id='standard-start-adornment'
+                  data-cy='lastName'
+                  name='lastName'
+                  className={styles.textField}
+                  variant='outlined'
+                  label={CHECKOUT_TEXT_FIELDS[language].lastName}
+                  value={values.lastName}
+                  onChange={handleChange}
+                  error={touched.code && !!errors.code}
+                />
+                {touched.code && errors.code && (
+                  <div data-cy='code-error' className={styles.error}>
+                    {errors.code}
+                  </div>
+                )}
+              </div>
+            </div>
+            <div className={styles.contactInfoFields}>
+              <TextField
+                size='small'
+                id='standard-start-adornment'
+                data-cy='email'
+                name='email'
+                className={styles.textField}
+                variant='outlined'
+                label={CHECKOUT_TEXT_FIELDS[language].email}
+                value={values.email}
+                onChange={handleChange}
+                error={touched.code && !!errors.code}
+              />
+              {touched.code && errors.code && (
+                <div data-cy='code-error' className={styles.error}>
+                  {errors.code}
+                </div>
+              )}
+              <TextField
+                size='small'
+                id='standard-start-adornment'
+                data-cy='phoneNumber'
+                name='phoneNumber'
+                className={styles.textField}
+                variant='outlined'
+                label={CHECKOUT_TEXT_FIELDS[language].contactPhoneNumber}
+                value={values.phoneNumber}
+                onChange={handleChange}
+                error={touched.code && !!errors.code}
+              />
+              {touched.code && errors.code && (
+                <div data-cy='code-error' className={styles.error}>
+                  {errors.code}
+                </div>
+              )}
+            </div>
+            <div className={styles.contactPaymentInfo}>
+              <h2
+                className={`${styles.contactInfoTitle} ${styles.paymentTitle}`}
+              >
+                {CHECKOUT_TITLES[language].payment}
+              </h2>
+              <FormControl variant='outlined' className={styles.formControl}>
+                <InputLabel variant='outlined'>
+                  {CHECKOUT_TEXT_FIELDS[language].paymentMethod}
+                </InputLabel>
+                <Select
+                  label={CHECKOUT_TEXT_FIELDS[language].paymentMethod}
```

Comment:
its better to set bigger width for this select 
![image](https://user-images.githubusercontent.com/36926670/107872444-268e6680-6eb3-11eb-915e-7be5450581e8.png)

Reply:
fixed